config/master/manifests.jsonnet

```diff
@@ -143,9 +144,14 @@ local awsnode = {
                 exec: {
                   command: ["/app/grpc-health-probe", "-addr=:50051"],
                 },
-                initialDelaySeconds: 35,
+                initialDelaySeconds: 1,
+              },
+              livenessProbe: {
+                exec: {
+                  command: ["/app/grpc-health-probe", "-addr=:50051"],
+                },
+                initialDelaySeconds: 60,
```

Comment:
The PR description says "Increase liveness polling period".  I think this PR increases the liveness _initial delay_ period however - which one was intended? (I suspect this should have been `periodSeconds=60`)

(Separately) jsonnet style tip:  You can "merge" values with `+`.  Since liveness/readiness probes are very similar, I would do this as:
```jsonnet
livenessProbe: self.readinessProbe + {
    initialDelaySeconds: 60, // If desired, but see above comment
    periodSeconds: 60,
},
```

Reply:
I'll fix the comment. I think, like you said in https://github.com/aws/amazon-vpc-cni-k8s/pull/1028#discussion_r441854922, that we should just increase the initial timeout, but keep the period at the default in order to catch issues that happen after startup faster. That should give the aws-node about 90 seconds to get started, after a 60 s startup delay and three 10 s liveness probes.